Source/Core/DolphinQt/NetPlay/NetPlaySetupDialog.cpp

```diff
@@ -254,6 +257,14 @@ void NetPlaySetupDialog::show()
 
 void NetPlaySetupDialog::accept()
 {
+  if (StringUTF8CodePointCount(m_nickname_edit->text().toStdString()) > NetPlay::MAX_NAME_LENGTH)
+  {
+    // hard-coded number for translation purposes, to the dismay of a developer
```

Comment:
Rather than snark, can your comment instead explain why we need to hardcode? %1 + .arg doesn't seem particularly infeasible.

Reply:
Yeah I suppose I can be more specific about it.